hax/hax/handler.py

```diff
@@ -92,6 +95,21 @@ def pull_msg():
                         # intermittent error gets resolved.
                         decorated = (repeat_if_fails(wait_seconds=5))(fn)
                         decorated(item.evt)
+                        event: ConfHaProcess = item.evt
+                        ha_event = pevent(event.chp_event)
+                        proc_type = ptype(event.chp_type)
+                        if proc_type == ptype.M0_CONF_HA_PROCESS_M0D:
+                            svc_status = ServiceHealth.UNKNOWN
```

Comment:
1. Perhaps it makes sense to extract the whole block under this `if` into a separate function.
2. The if/elif can be simplified like this:
```python
status_map = {pevent.M0_CONF_HA_PROCESS_STOPPED: ServiceHealth.FAILED, pevent.M0_CONF_HA_PROCESS_STARTED: ServiceHealth.OK}
svc_status = status_map.get(ha_event, ServiceHealth.UNKNOWN)
```

Reply:
Agree. Thanks.